### PR TITLE
Upgraded HSQL version to 2.7.1

### PR DESCRIPTION
**Pull request by MarekB01**
https://github.com/BroadleafCommerce/QA/issues/4813

Updated hsql to version 2.7.1 Also had to add classifier parameter to hsql dependency, because by default it needs JDK-11

### CODE REVIEW

**Review by oleksiimiroshnyk — APPROVED** [review]

File: admin/broadleaf-open-admin-platform/pom.xml [hunk 1]
```diff
@@ -163,6 +163,8 @@
         <dependency>
             <groupId>org.hsqldb</groupId>
             <artifactId>hsqldb</artifactId>
+            <version>2.7.1</version>
```
Comment (riteshadhikari17) [inline admin/broadleaf-open-admin-platform/pom.xml]: No need to define version on each of the POMs. It should come from parent POM. Not sure about the classifier so maybe test without it first.

File: common/pom.xml [hunk 2]
```diff
@@ -436,6 +436,8 @@
         <dependency>
             <groupId>org.hsqldb</groupId>
             <artifactId>hsqldb</artifactId>
+            <version>2.7.1</version>
```
Comment (riteshadhikari17) [inline common/pom.xml]: Same as above

File: integration/pom.xml [hunk 3]
```diff
@@ -214,6 +214,8 @@
         <dependency>
             <groupId>org.hsqldb</groupId>
             <artifactId>hsqldb</artifactId>
+            <version>2.7.1</version>
```
Comment (riteshadhikari17) [inline integration/pom.xml]: same as above

**Review by riteshadhikari17 — CHANGES_REQUESTED** [review]

**Comment by danielcolgrove** [comment]
The classifier should be in the project pom (not the parent pom).  Only the version would be in the parent pom.  The project pom can decide if it needs a classifier which we should use classifier as needed.

**Comment by riteshadhikari17** [comment]
@danielcolgrove Classifier is needed for us because we use jdk8 when building. These are needed for ITs only. What if we add test scope to all ? Like this
```
  <dependency>
      <groupId>org.hsqldb</groupId>
      <artifactId>hsqldb</artifactId>
      <classifier>jdk8</classifier>
      <scope>test</scope>
  </dependency>
```

**Comment by danielcolgrove** [comment]
Using the scope should be fine when the scope is needed - for "test" is a good example of when you would want it.

**Review by riteshadhikari17 — APPROVED** [review]

**Pull request merged**